pkg/controller/direct/bigquerydataset/dataset_controller.go

```diff
@@ -151,6 +151,14 @@ func (a *Adapter) Create(ctx context.Context, createOp *directbase.CreateOperati
 		desiredDataset.DefaultEncryptionConfig.KMSKeyName = kmsRef.External
 	}
 	dsHandler := a.gcpService.DatasetInProject(a.id.Parent().ProjectID, a.id.ID())
+	// var infor string
```

Comment:
Nit: we should not add commented code unless there's a reason (and I think this is debug code?)